flepimop/gempyor_pkg/src/gempyor/utils.py

```diff
@@ -447,6 +536,9 @@ def rolling_mean_pad(
 
 
 def print_disk_diagnosis():
+    """
+    Reads and prints AWS diagnoses. Includes total bytes, used bytes, free bytes.
```

Comment:
Titles should be one sentence usually, perhaps something like "Reads and print AWS disk diagnostic information." or similar would make a good title and the rest of the information could go into the description.